x-pack/plugins/elastic_assistant/server/routes/utils.ts

```diff
@@ -170,6 +170,7 @@ export const getLlmType = (actionTypeId: string): string | undefined => {
   const llmTypeDictionary: Record<string, string> = {
     [`.gen-ai`]: `openai`,
     [`.bedrock`]: `bedrock`,
+    [`.gemini`]: `gemini`,
```

Comment:
can you make this part of the second PR please?

Reply:
fixed